src/drivers/intel/dmic/dmic.c

```diff
@@ -248,6 +260,16 @@ static int dmic_set_config(struct dai *dai, struct ipc_config_dai *common_config
 
 	dcache_writeback_region(dmic_prm[di], sizeof(struct sof_ipc_dai_dmic_params));
 	ret = dmic_set_config_computed(dai);
+
+#elif CONFIG_INTEL_DMIC_NHLT
+	ret = dmic_set_config_nhlt(dai);
```

Comment:
can we can the name to dmic_set_config_blob ? since blob data is not always from nhlt and we may generate it by other tools and put it in topology.

Reply:
yes, I would also like some another name. This is not actually any nhlt standardized blob, it is intel specific struct which is not described too much anywhere. The special blob is just embedded as data inside the actual nhlt. like binary blob in alsa topology.

Reply:
Yes, it's a binary registers dump and not covered by NHLT spec but maybe that would make too long function name. I can try use "blob" here, let's see.